docs/reviews/2026-04-21/13-39-md-review.md

```
@@ -0,0 +1,82 @@
+# MD 정합성 리뷰 — 2026-04-21 13-39
+
+## 대상
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)